test/stdlib/StringIndex.swift

```diff
@@ -1060,7 +1060,8 @@ suite.test("String.replaceSubrange index validation")
   }
 }
 
-suite.test("Substring.removeSubrange entire range") {
+suite.test("Substring.removeSubrange entire range")
+.xfail(.always("Disabled due to rdar:\/\/99627624")) {
```

Comment:
Isn't this invalid escape sequences?

Reply:
Oh my editor turned this (//) into a comment and I thought it was illegal. Thanks 👍